packages/bbui/src/Form/Select.svelte

```diff
@@ -13,6 +13,7 @@
   export let options = []
   export let getOptionLabel = option => extractProperty(option, "label")
   export let getOptionValue = option => extractProperty(option, "value")
+  export let getOptionIcon = option => extractObjectProperty(option, "icon")
```

Comment:
I think we can actually simplify this, because if the option is a string then we don't want an icon at all, so we could maybe change this to 
```js
export let getOptionIcon = option => option?.icon
```
Which just mean it will default to the `icon` property of options if they're objects, otherwise have no icon. I think that works for the default behaviour. It's different than `value` and `label` because both of those need to be shown at all times.